app/src/main/java/org/thoughtcrime/securesms/media/MediaPage.kt

```diff
@@ -153,12 +153,15 @@ private fun ThumbnailRow(
                             it.diskCacheStrategy(DiskCacheStrategy.NONE)
                         }
                     } else if (item.hasPlaceholder) {
-                        Image(
-                            painter = painterResource(item.placeholder(LocalContext.current)),
-                            contentDescription = null,
-                            modifier = Modifier.fillMaxSize(),
-                            contentScale = ContentScale.Inside
-                        )
+                        val placeholder = item.placeholder(LocalContext.current)
+                        if (placeholder != 0) {
```

Comment:
Isn't that what the `hasPlaceholder` is supposed to check?

Reply:
No..it is a different check. And for some reason one of the placeholder is missing somewhere.